tests/network/sriov.go

```diff
@@ -172,36 +172,28 @@ var _ = Describe("SRIOV", Serial, decorators.SRIOV, func() {
 			if vmi.Annotations == nil {
 				vmi.Annotations = make(map[string]string)
 			}
+			const (
+				specialNetTag = "specialNet"
+				pciAddress    = "0000:08:00.0"
+			)
 			vmi.Annotations[v1.InstancetypeAnnotation] = testInstancetype
 
 			for idx, iface := range vmi.Spec.Domain.Devices.Interfaces {
 				if iface.Name == sriovnet1 {
-					iface.Tag = "specialNet"
+					iface.Tag = specialNetTag
+					iface.PciAddress = pciAddress
 					vmi.Spec.Domain.Devices.Interfaces[idx] = iface
 				}
 			}
```

Comment:
Please consider replacing this loop with an interface definition on the vmi specification above.